src/Inkluzitron/Utilities/FormatByValue.cs

```diff
@@ -33,12 +34,38 @@ public string ToString(string format, IFormatProvider formatProvider = null)
                 };
             }
 
-            return ValueNumber.Value switch
+            if (options.Length == 3 && options[0] != "ext")
```

Comment:
hmm ext??? what?

Reply:
Janchovo "staré" formátování používá např. `{0:a:b:c}` a bylo potřeba nějak rozlišit, jestli se má formátovat starým nebo novým způsobem. Plácnul jsem tam "ext" jako rozlišovadlo.